.github/workflows/jit.yml

```
@@ -94,35 +94,40 @@ jobs:
       - name: Native Windows
         if: runner.os == 'Windows' && matrix.architecture != 'ARM64'
         run: |
-          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}
+          choco upgrade llvm -y
+          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }} --debug --verbose --noop 
```

Mind explaining the addition of these options?

```suggestion
          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}
```